Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.15.4
+FROM alpine:3.16.1
```

Comment:
(not related to the PR)
Sometimes we pin the patch and sometimes we do not, we should at some point agree on one approach.